src/components/App/App.js

```diff
@@ -1,56 +1,56 @@
 import "./App.css";
 import { useState, useEffect } from "react";
-// import movieData from "../../mockMovies";
 import Header from "../Header/Header";
 import MainContentWrapper from "../MainContentWrapper/MainContentWrapper";
 import MovieCards from "../MovieCards/MovieCards";
 // import SingleMovieDetails from "../SingleMovieDetails/SingleMovieDetails";
-import { getAllMovies } from "../../apiCalls/apiCalls";
+import { getAllMovies, getIndividualMovie } from "../../apiCalls/apiCalls";
+import SingleMovieDetails from "../SingleMovieDetails/SingleMovieDetails";
 
 function App() {
   const [fullData, setFullData] = useState([]);
```

Comment:
I love the way you're cleaning and filtering your data after you fetch it!  Are you planning to ever use the fullData that you're storing in state?  If not, do you even need to bother storing that?